samples/wifi/scan/prj.conf

```diff
@@ -20,7 +20,7 @@ CONFIG_NET_OFFLOAD=y
 CONFIG_INIT_STACKS=y
 
 # Memories
-CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_MAIN_STACK_SIZE=4096
```

Comment:
This looks like a fixup commit please fixup/squash to original commit that introduced multiple scans